packages/darklang/stdlib/httpclient.dark

```diff
@@ -42,23 +42,16 @@ module Darklang =
         ("authorization", ("bearer " ++ token))
 
 
-      // TODOs:
-      // - maybe the below should be in another ContentType module,
-      //   accessible by e.g. `HttpClient.ContentType.form`?
-      // - these should likely be Constants (rather than fns)
 
-      /// Returns a header <type (String*String))> with {{content-type}} set for HTML form requests or responses
-      let formContentType () : (String * String) =
-        ("content-type", "application/x-www-form-urlencoded")
+      module ContentType =
+        /// A header <type (String*String))> with {{content-type}} set for HTML form requests or responses
+        let form = ("content-type", "application/x-www-form-urlencoded")
 
-      /// Returns a header <type (String*String))> with {{content-type}} set for JSON requests or responses
-      let jsonContentType () : (String * String) =
-        ("content-type", "application/json; charset=utf-8")
+        /// A header <type (String*String))> with {{content-type}} set for JSON requests or responses
+        let json = ("content-type", "application/json; charset=utf-8")
```

Comment:
I think this should be just "application/json" with no charset - I think we've been making that mistake for years.